### PR TITLE
bluetoothctl: add power and help examples

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

File: pages/linux/bluetoothctl.md [hunk 1]
```diff
@@ -7,22 +7,30 @@
...
+`bluetoothctl connect {{mac_address}}`
 
-- Connect a paired device:
+- Disconnect from a paired device:
 
-`bluetoothctl -- connect {{mac_address}}`
+`bluetoothctl disconnect {{mac_address}}`
 
-- Disconnect a paired device:
+- Show help for `bluetoothctl`:
```
Comment (CleanMachine1) [inline pages/linux/bluetoothctl.md:34]:
```suggestion
- Display help:
```

File: pages/linux/bluetoothctl.md [hunk 2]
```diff
@@ -7,22 +7,30 @@
 
 `bluetoothctl`
 
-- List devices:
+- List all known devices:
```
Comment (CleanMachine1) [inline pages/linux/bluetoothctl.md:10]:
```suggestion
- List all known and available devices:
```

**Comment by CleanMachine1** [comment]
Should the B's in bluetooth be capitalized since it is a standard and a trademark?

File: pages/linux/bluetoothctl.md (hunk 2 above)
Comment (marchersimon) [reply pages/linux/bluetoothctl.md:10]: It also shows not available devices

File: pages/linux/bluetoothctl.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/bluetoothctl.md:34]: @marchersimon Could you apply this so we can merge?

**Review by navarroaxel — APPROVED** [review]

**Review by Waples — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**